File: Python/Manual 01/Question04.py
a = int(input("Person A, Enter your age: "))
b = int(input("Person B, Enter your age: "))
c = int(input("Person C, Enter your age: "))

if a > b and a > c:
    print("Person A is the oldest and")
elif b > a and b > c:
    print("Person B is the oldest and")
else:
    print("Person C is the oldest and")

if a < b and a < c:
    print("Person A is the youngest")
if b < a and b < c:
    print("Person B is the youngest")
if c < a and c < b:
    print("Person C is the youngest")

File: JavaScript/Manual 01/Question04.js
const readline = require('readline');

const rl = readline.createInterface({
    input: process.stdin,
    output: process.stdout
});

rl.question("Enter the age of Person A: ", function (ageA) {
    rl.question("Enter the age of Person B: ", function (ageB) {
        rl.question("Enter the age of Person C: ", function (ageC) {

            if (ageA > ageB && ageA > ageC)
                console.log("Person A is the oldest.");
            else if (ageB > ageA && ageB > ageC)
                console.log("Person B is the oldest.");
            else if (ageC > ageA && ageC > ageB)
                console.log("Person C is the oldest.");
            else
                console.log("There are multiple people with the same age.");

            if (ageA < ageB && ageA < ageC)
                console.log("Person A is the youngest.");
            else if (ageB < ageA && ageB < ageC)
                console.log("Person B is the youngest.");
            else if (ageC < ageA && ageC < ageB)
                console.log("Person C is the youngest.");
            else
                console.log("There are multiple people with the same age.");
            rl.close();
        });
    });
})